.github/release-drafter.yml

```diff
@@ -15,16 +15,19 @@ categories:
     label: 'deprecation'
 exclude-labels:
   - 'skip-changelog'
-change-template: '* $TITLE (#$NUMBER)'
+category-template: '### $TITLE'
+change-template: '* $TITLE [#$NUMBER]($URL)'
 template: |
-  [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
+  ## Release v$NEXT_PATCH_VERSION (20YY/MM/DD)
 
-  ## Highlights
+  [![Digital Object Identifier for PyGMT v$NEXT_PATCH_VERSION](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
```

Comment:
I think `DOI` is well known.
```suggestion
  [![DOI for PyGMT v$NEXT_PATCH_VERSION](https://zenodo.org/badge/DOI/10.5281/zenodo.3781524.svg)](https://doi.org/10.5281/zenodo.3781524)
```

Reply:
Prefer to keep this alt-text in long form for accessibility (e.g. those using screen readers). Also, the previous ReST alt-text used "Digital Object Identifier" too.

Reply:
OK with the long-form.